poll/public/css/poll.css

```diff
@@ -146,7 +146,7 @@ li.poll-result .poll-image {
     border-top: 0;
 }
 
-.survey-table tr:last-child td {
+.survey-table > tbody > tr:last-child td {
```

Comment:
The `>` is not strictly necessary, but the `tbody` is.  Otherwise this rule will apply to thead, which we don't want.